Doc/library/pdb.rst

```diff
@@ -61,6 +61,11 @@ useful than quitting the debugger upon program's exit.
    :file:`pdb.py` now accepts a ``-c`` option that executes commands as if given
    in a :file:`.pdbrc` file, see :ref:`debugger-commands`.
 
+.. versionadded:: 3.7
+   :file:`pdb.py` now accepts a ``-m`` option that execute modules similar to how
+   ``python3 -m`` does. The debugger will stop in the first line like with a script.
```

Comment:
Slight rephrasing of the second sentence here: `As with a script, the debugger will pause execution just before the first line of the module.`